### PR TITLE
qa/rgw: reenable wait-for-scrub

### PR DESCRIPTION
**Pull request by cbodley**
let the osds finish a deep scrub at the end of rgw tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by cbodley** [comment]
cc @neha-ojha @jdurgin @vumrao

**Review by jdurgin — APPROVED** [review]
excellent!

**Comment by vumrao** [comment]
Thank you, Casey.

**Comment by neha-ojha** [comment]
> cc @neha-ojha @jdurgin @vumrao

@cbodley how are the teuthology runs looking with this change? I'd be curious to know how much time it is adding to the tests.

**Comment by cbodley** [comment]
@neha-ojha still trying to get a clean qa run, but the runtimes of the passing jobs in https://pulpito.ceph.com/cbodley-2022-04-18_18:25:44-rgw-wip-cbodley-testing-distro-default-smithi/ look comparable to previous runs 👍 once i get full results, i'll try to do a more detailed comparison

**Comment by cbodley** [comment]
rebased, and added a commit that runs `radosgw-admin gc process` to avoid scrubbing all of rgw's deleted objects

**Comment by cbodley** [comment]
@ljflores @yuriw note that this only touches the rgw suite, so doesn't need any rados testing. i'm happy to handle the qa

**Comment by ljflores** [comment]
@cbodley sure, we ran a rados suite and I don't see any issues, so go ahead with the rgw QA!

**Comment by ljflores** [comment]
@yuriw don't merge this PR, as Casey is doing individual tests.

**Comment by cbodley** [comment]
@soumyakoduri testing this PR uncovered a crash in dbstore: https://tracker.ceph.com/issues/59216

do you want to try to fix that before we merge this PR, or would you prefer to follow up later?

otherwise passed qa in https://pulpito.ceph.com/cbodley-2023-03-23_13:55:57-rgw-main-distro-default-smithi/

**Comment by soumyakoduri** [comment]
> @soumyakoduri testing this PR uncovered a crash in dbstore: https://tracker.ceph.com/issues/59216
> 
> do you want to try to fix that before we merge this PR, or would you prefer to follow up later?
> 

Thanks Casey. Please go ahead and merge this PR. I will look into it asap.

**Pull request merged**
